src/curl.cpp

```diff
@@ -3529,6 +3580,9 @@ int S3fsCurl::ListBucketRequest(const char* tpath, const char* query)
   curl_easy_setopt(hCurl, CURLOPT_URL, url.c_str());
   curl_easy_setopt(hCurl, CURLOPT_WRITEDATA, (void*)&bodydata);
   curl_easy_setopt(hCurl, CURLOPT_WRITEFUNCTION, WriteMemoryCallback);
+  if(S3fsCurl::is_verbose){
+    curl_easy_setopt(hCurl, CURLOPT_DEBUGFUNCTION, S3fsCurl::CurlDebugBodyInFunc);     // replace debug function
```

Comment:
Should this call the following as above:

```cpp
curl_easy_setopt(hCurl, CURLOPT_VERBOSE, true);
```

Repeated below.

Reply:
CURLOPT_VERBOSE is set in ResetHandle() called from CreateCurlHandle method.
Here we are replacing(Resetting) the default CurlDebugFunc handler set by ResetHandle() with CurlDebugBodyInFunc.
So this is no problem.